core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/parameter/network/AzureNetworkV4Parameters.java

```diff
@@ -81,6 +92,7 @@ public Map<String, Object> asMap() {
         putIfValueNotNull(map, "networkId", networkId);
         putIfValueNotNull(map, "subnetId", subnetId);
         putIfValueNotNull(map, "databasePrivateDsZoneId", databasePrivateDnsZoneId);
+        putIfValueNotNull(map, "aksPrivateDsZoneId", aksPrivateDnsZoneId);
```

Comment:
also, all these random string keys might come from an enum, instead

Reply:
For now I move them to a Constant class in project common, so that it can be used across the whole cloudbreak project.